### DISCUSSION TITLE
`[blank]` visible in some outputs when NO_COLOR is set

### DISCUSSION THREAD

**Discussion by tuurep** [Slackadays/Clipboard#109] [General]
Hello, I would have sent a simple PR but I'm not quite sure how to fix after looking at it.

I found two contexts where `[blank]` gets printed:

```bash
NO_COLOR="Yup" cb copy "123"
```

Outputs `✅ Copied text "123"[blank]`

```bash
NO_COLOR="Yup" cb show
```

Outputs `• Here are the first 3 bytes in clipboard 0: [blank]123`

Note: `cb copy` doesn't show the `[blank]` when copying something other than a string, but `cb show` will show it regardless of what's in the clipboard

**Comment by Slackadays** [comment]
@tuurep Fixed back in https://github.com/Slackadays/Clipboard/commit/f41fc4872b557df0a64c8454898ada10779b7149

**Comment by tuurep** [comment]
Ah, I see. Thank you :)
